src/utils/schema.ts

```diff
@@ -1,17 +1,21 @@
-export interface Image {
+export type Image = {
   src: string;
   alt: string;
-}
+};
 
-export interface Member {
+export type Member = {
   name: string;
   picture: Image;
   role: RoleName;
   linkedin: string;
   active: boolean;
   graduationYear: number;
   testimonial?: string;
-}
+};
+
+export type TestimonialMember = Member & {
+  testimonial: string;
+};
```

Comment:
Going off of the `interface` vs. `type` comment above, this could be done as `export interface TestimonialMember extends Member {...`

Reply:
Will update this to reflect a decision on the above.